libs/libcommon/src/libcommon/utils.py

```diff
@@ -83,3 +84,9 @@ def inputs_to_string(
     if prefix is not None:
         result = f"{prefix},{result}"
     return result
+
+
+def is_image_url(text: str) -> bool:
+    is_url = text.startswith("https://") or text.startswith("http://")
+    (mime_type, _) = mimetypes.guess_type(text)
```

Comment:
small suggestion: strip query parameters in URLs:

```suggestion
    (mime_type, _) = mimetypes.guess_type(text.split("/")[-1].split("?")[0])
```

Reply:
Thanks!